.travis.yml

```diff
@@ -3,15 +3,19 @@ language: python
 
 services: rabbitmq
 
-python:
-  - '2.7'
-
 install:
   - pip install tox
 
-env:
-  - TOX_ENV=py33
-  - TOX_ENV=py34
+matrix:
+  include:
+    - python: 3.4
+      env: TOX_ENV=py34-test
+
+    - python: 3.5
+      env: TOX_ENV=py35-test
+
+    - python: 3.6
```

Comment:
Should we also add python 3.7 here?

Reply:
I tried, but couldn't get it to work correctly.
Python 3.7 is not available on Travis's default Ubuntu Trusty images, only on Xenial. But the Xenial images don't have rabbitmq installed. I tried to manually install rabbit, but still couldn't get the tests to work with it, so I gave up.